tutor/src/api/index.coffee

```diff
@@ -158,7 +158,7 @@ startAPI = ->
 
 
   # connectRead(ReferenceBookPageActions, pattern: 'pages/{id}', trigger: 'loadSilent', handledErrors: ['*'])
-  # connectRead(ReferenceBookExerciseActions, url: (url) -> url)
+  connectRead(ReferenceBookExerciseActions, url: (url) -> url)
 
```

Comment:
Sorry, `git blame` says this was me, but I have no recollection of why I did it ☹️